src/chrome/content/rules/HubSpot.xml

```diff
@@ -45,6 +41,7 @@ Non-2xx HTTP code: http://forums.hubspot.com/ (200) => https://hubspot3.vanillaf
 
 	² Refused
 	* 503, akamai
+	⁴ 404
```

Comment:
`hubspot3.vanillaforums.com` is a 404, but `forums.hubspot.com` serves a mismatching cert.